es-models/gdc_from_graph/descriptions.yaml

```diff
@@ -0,0 +1,3234 @@
+_meta:
+  descriptions:
+    annotations.aliquot.aliquot_quantity: 'The quantity in micrograms (ug) of the
+      aliquot(s) derived from the analyte(s) shipped for sequencing and characterization.
+
```

Comment:
wondering about these extra newlines...

Reply:
yeah, `yaml` was spitting them out like that. did a `strip` before dumping